packages/saber/lib/cli.js

```diff
@@ -83,6 +73,34 @@ cli.help()
 
 cli.parse()
 
+function startServer({ cwd, options, host, port = 3000, lazy }) {
+  return new Promise((resolve, reject) => {
+    require('..')(Object.assign({ cwd, dev: true }, options), {
+      server: {
+        host,
+        port
+      },
+      build: {
+        lazy
+      }
+    })
+      .serve()
```

Comment:
we can use https://github.com/sindresorhus/get-port instead, `.serve` has side effects, it runs `.run` before creating the server.